src/test/java/com/bobocode/hoverla/bring/testsubject/beandefinition/classbased/ClassBasedBeanDefinitionWithName.java

```diff
@@ -0,0 +1,11 @@
+package com.bobocode.hoverla.bring.testsubject.beandefinition.classbased;
+
+import com.bobocode.hoverla.bring.annotation.Bean;
+import lombok.EqualsAndHashCode;
+
+@Bean(name = ClassBasedBeanDefinitionWithName.BEAN_DEFINITION_WITH_NAME)
+@EqualsAndHashCode
```

Comment:
We don't expect users to implement equals and hashCode for their class based beans